pkg/kclient/supervisord.go

```diff
@@ -0,0 +1,71 @@
+package kclient
+
+import corev1 "k8s.io/api/core/v1"
+
+const (
+	// The init container name for supervisord
+	supervisordInitContainerName = "copy-supervisord"
+
+	// The default image for odo init bootstrapper container
+	defaultBootstrapperImage = "jeevandroid/odo-init-image"
```

Comment:
Most of the constants here are already defined in `occlient.go` (https://github.com/openshift/odo/blob/a8a5eca39b80abf693bbe1e5e98880f9f31bf34a/pkg/occlient/occlient.go#L109)

We should probably abstract it away to one common place and use the same const everywhere. Or just export it `occlient` and use it here for now, but having it defined in two places will get messy and prone to bugs :-/